packages/commonwealth/client/scripts/views/pages/notifications/LegacyNotificationsContent.tsx

```diff
@@ -1,76 +0,0 @@
-import { byDescendingCreationDate } from 'helpers';
-import React, { useState } from 'react';
-import { Virtuoso } from 'react-virtuoso';
-import app from 'state';
-import { CWText } from 'views/components/component_kit/cw_text';
-import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
-import { NotificationRow } from 'views/pages/notifications/notification_row';
-
-export const LegacyNotificationsContent = () => {
-  const [allRead, setAllRead] = useState<boolean>(false);
-
-  const discussionNotifications =
-    app.user.notifications.discussionNotifications;
-  const chainEventNotifications =
-    app.user.notifications.chainEventNotifications;
-
-  const mostRecentFirst = [
-    ...discussionNotifications.concat(chainEventNotifications),
-  ].sort(byDescendingCreationDate);
-
-  return (
-    <div className="NotificationsPage">
```

Comment:
remove `notifications/index.scss`
The file is imported but none of the classes from this file are used in the codebase